Caboodle/Sms/Sms.shared.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Sms
+    {
+        public static bool CanSendSms => GetCanSendSms();
+
+        public static bool CanSendSmsInBackground => GetCanSendSmsInBackground();
+
+        public static void SendSms(string recipient = null, string message = null)
```

Comment:
I think these two methods should be combined into one that takes an enum that specifies whether to send in background or not. Something along the lines of this:
```csharp
enum SmsSendType {
    Foreground,
    Background,
    // if the device supports background sending, the use that, otherwise use foreground
    PreferBackground
}
```

Reply:
@mattleibow  I think this is simple and have self explanatory method names. But I can make changes if you want it that way,